packages/apps/graph/package.json

```diff
@@ -1,14 +1,24 @@
 {
   "name": "@kadena/graph",
   "version": "1.0.6",
-  "private": true,
-  "description": "",
-  "keywords": [],
-  "license": "ISC",
-  "author": "",
-  "main": "index.js",
+  "description": "GraphQL server, available for running your own GraphQL endpoint. This project uses chainweb-data as the datasource.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kadena-community/kadena.js.git",
+    "directory": "packages/apps/graph"
+  },
+  "license": "BSD-3",
+  "contributors": [
+    "Albert Groothedde <albert@kadena.io>",
+    "Maarten van den Hoven <maarten.vandenhoven@deptagency.com>",
+    "Nil Amrutlal <nil.amrutlal@deptagency.com>"
+  ],
+  "bin": {
+    "kadena-graph": "./dist/index.js"
+  },
   "scripts": {
-    "build": "pnpm run pactjs:generate:contract:coin && pnpm prisma:generate",
+    "prebuild": "pnpm run pactjs:generate:contract:coin && pnpm prisma:generate",
```

Comment:
Confirm this change with @alber70g since I believe it was like this but changed for some reason I cant recall rn

Reply:
I think turbo doesn't execute `prebuilt`s. Maybe it was a problem with Rush, and isn't an issue anymore. Just check in CI whether it executes